MonoToMicroAssets/MonoToMicroCF.template

```diff
@@ -367,7 +367,7 @@
                     "configure" : {
                         "commands" : {
                             "0-clone-repo" : { "command" :
-                                "git clone https://github.com/aws-samples/unishop-monolith-to-microservices.git /home/ec2-user/MonoToMicro"
+                                "git clone https://github.com/dmahapatro/unishop-monolith-to-microservices.git /home/ec2-user/MonoToMicro"

```

Comment:
@deki Assume this is just for testing purposes?

Reply:
yes, will fix it before merging